Tests/RockLib.Configuration.ObjectFactory.Tests/.editorconfig

```diff
@@ -0,0 +1,68 @@
+root = true
+
+[*]
+end_of_line = crlf
```

Comment:
Same as the other project.  This should probably be changed to `lf`.

Reply:
Done